examples/incompressible-flow/3d-taylor-green-vortex/plot_dissipation_rate.py

```diff
@@ -27,6 +27,8 @@
 parser.add_argument("-ens", "--enstrophy", type=str, help="Name of the input file for the enstrophy", required=True)
 
 parser.add_argument("-v", "--viscosity", type=float, help="viscosity", required=True)
+parser.add_argument("--validate", action="store_true", help="Launches the script in validation mode. This will log the content of the graph and prevent the display of figures", default=False)
```

Comment:
Maybe in all the validation test, the keyword for the validation should only be "--validate" and never just "-v" so it doesn't get confusing here. Also, everything will be standardize and we will avoid potential argparse problem in the future.

Reply:
Agreed. Let's keep --validate because anyway users won't use that flag.